internal/cmd/login.go

```diff
@@ -456,8 +460,8 @@ func deviceFlowLogin(ctx context.Context, client *api.Client, cli *CLI) (*api.Lo
 	url := resp.VerificationURI + "?code=" + resp.UserCode
 
 	// display to user
-	cli.Output("Navigate to " + url + " and verify your code:\n")
-	cli.Output("\t\t" + resp.UserCode + "\n")
+	fmt.Fprintf(os.Stderr, "  Navigate to %s and verify your code:\n\n", termenv.String(url).Underline().String())
+	fmt.Fprintf(os.Stderr, "\t\t%s\n\n", termenv.String(resp.UserCode).Bold().String())
```

Comment:
This adds highlighting to the output to more easily identity important components

Reply:
What does this look like? Feels like a lot of new lines.

Reply:
It's the same content but with some highlights. Will add some screenshots to PR description